### PR TITLE
Implement transaction hash row for new transaction details sheet

### PR DESCRIPTION
**Pull request by jkadamczyk**
Fixes APP-34

## What changed (plus any additional context for devs)

* Implemented a transaction hash row for the upcoming new Transaction Details sheet

**More details in Linear**

### CODE REVIEW

File: src/screens/transaction-details/ThinSymbolRow.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Box, Columns, Inline, Text } from '@/design-system';
+
+type Props = {
+  icon: string;
+  title: string;
+  value: string;
+};
+
+export const ThinSymbolRow: React.FC<Props> = ({ icon, title, value }) => {
```
Comment (skylarbarrera) [inline src/screens/transaction-details/ThinSymbolRow.tsx:10]: not big on this component name, something like TransactionDetailsRow makes more sense to me but your call

**Review by skylarbarrera — APPROVED** [review]

File: src/screens/transaction-details/ThinSymbolRow.tsx (hunk 1 above)
Comment (jkadamczyk) [reply src/screens/transaction-details/ThinSymbolRow.tsx:10]: Wel, there will be a different kind of row, hence the Thin, but good to know I will figure out a different name for that

File: src/screens/transaction-details/__tests__/shortenTxHashString.test.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { shortenTxHashString } from '@/screens/transaction-details/shortenTxHashString';
+
+const exampleTransactionHash =
+  '0x30f2e457cd93d7091bc3acefe82d091363dfb00b0e6d2b3fefdbda4c6031d2a8';
+
+test('converting long transaction hash into short ellipsized in the middle with 6 chars on each side', () => {
+  expect(shortenTxHashString(exampleTransactionHash)).toEqual(
+    '0x30f2...31d2a8'
+  );
+});
```
Comment (estrattonbailey) [inline src/screens/transaction-details/__tests__/shortenTxHashString.test.ts:10]: 😍

**Review by estrattonbailey — APPROVED** [review]

File: src/screens/transaction-details/TransactionDetailsContent.tsx [hunk 3]
```diff
@@ -1,17 +1,37 @@
...
-      <Text color="label" size="11pt">
-        {JSON.stringify(transaction, null, 2)}
-      </Text>
+    <Box
+      background="surfacePrimary"
+      flexGrow={1}
+      paddingHorizontal="20px"
+      paddingBottom="20px"
+    >
+      <Divider />
```
[the remark this answers is not in the document]
Comment (jkadamczyk) [reply src/screens/transaction-details/TransactionDetailsContent.tsx:29]: Nope, there will be like only 3 of them so I will add them manually in the main component

**Pull request merged**